postgres/osfamilymap.yaml

```
@@ -62,6 +62,10 @@ RedHat:
     gpgkey: 'https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-{{ release }}'
     baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-$releasever-$basearch'
 
+{%- if 'osfinger' in grains and grains.osfinger in ('CentOS Linux-7', 'CentOS Linux-8') %}
+  pkg_python: python3-psycopg2
```

And set the default to `python3-psycopg2` as it was in you initial implementation, since it would cover modern Ubuntus too.
```suggestion
  pkg_python: python-psycopg2
```

---

It is  `python3-psycopg2` for Centos 7.  Maybe `{%- if grains.get('osmajorrelease') >= 7) %}` should wrap this?

---

Oh, yeah. It would even better.